Sources/GraphQLKit/Graphiti+Router.swift

```diff
@@ -3,8 +3,8 @@ import Graphiti
 import GraphQL
 
 extension RoutesBuilder {
-    public func register<RootType>(graphQLSchema schema: Schema<RootType, Request>, withResolver rootAPI: RootType, at path: PathComponent="graphql") {
-        self.post(path) { (request) -> EventLoopFuture<Response> in
+    public func register<RootType>(graphQLSchema schema: Schema<RootType, Request>, withResolver rootAPI: RootType, at path: PathComponent="graphql", postBody: HTTPBodyStreamStrategy = .collect) {
```

Comment:
To make it more clear what this parameter is doing I personally would prefer it if it's more verbose, like `postBodyStreamStrategy`. What do you think.